R/clean.R

```diff
@@ -29,14 +29,11 @@ clean <- function(path = ".") {
   )
 
   if (!file.exists(toml_path)) {
-    ui_throw(
+    cli::cli_abort(c(
```

Comment:
I suppose `cli::` is unnecessary because this package imports cli.

https://github.com/extendr/rextendr/blob/232133b32125d2f82748668f6cabe64cddb0a1b5/DESCRIPTION#L45

Reply:
In general, I think it's best to fully qualify functions from the namespace. 

https://r-pkgs.org/dependencies-in-practice.html#sec-dependencies-in-imports-r-code

> Our recommended default is to call external functions using the package::function() syntax:

Reply:
Also having a package in the Imports doesn't automatically load or attach the package. We could `@import cli` using roxygen but we aren't currently doing so, so `::` isn't unnecessary anyway.